data/README.md

```diff
@@ -0,0 +1,19 @@
+# Data
+
+The [`multiplier/`](multiplier/) folder contains the same data generated by the [MultiPLIER project](https://github.com/greenelab/multi-plier) but separated by model matrices so they can be more easily downloaded.
+Please, if you use any of these MultiPLIER files, cite the corresponding paper [here](https://doi.org/10.1016/j.cels.2019.04.003).
+All the files were downloaded from [this figshare archive](https://doi.org/10.6084/m9.figshare.5716033.v4), and later processed by the notebooks in [this folder](https://github.com/greenelab/phenoplier/tree/master/nbs/01_preprocessing).
+
+* `clustering/cluster_lvs/`: contains the set of LVs that distinguish traits in cluster (separated by partitions) from the rest.
```

Comment:
Should "in cluster" be something like "in each cluster"?

Reply:
Yes, good catch!